test/core/error-tracker.test.ts

```diff
@@ -0,0 +1,563 @@
+/* eslint-disable no-multi-spaces */
+import { createErrorTracker } from '../../packages/basic-crawler/src/internals/error-tracker';
+
+const g = (error: { name?: string; message?: string; code?: string; stack?: string }) => {
+    return {
+        name: Math.random().toString(36).slice(2),
```

Comment:
not tthat it would matter much for tests, but we can use `crypto.randomUUID()` instead